src/content/en/fundamentals/performance/prpl-pattern/index.md

```diff
@@ -221,6 +222,12 @@ The combination of HTTP/2 and HTTP/2 server push provides the _benefits_ of
 bundling (reduced latency) without actual bundling. Keeping resources separate
 means they can be cached efficiently and be shared between pages.
 
+HTTP/2 Push needs to be utilized with care, as it forces data to the browser,
```

Comment:
sooo there is even more nuance here. If you are following PRPL 100% you will have a sw which will intercept any future connections to the server so you will actually not be pushing since there is no open h2 connection to say index.html